e2e-tests/portal/cypress/support/portal-api/api.js

```diff
@@ -190,6 +190,19 @@ module.exports.insertGefApplication = (deal, token) => cy.request({
   return resp.body;
 });
 
+exports.setGefSubmissionType = (dealId, payload, token) => cy.request({
```

Comment:
Isn't this a general "update GEF" endpoint? Could be `updateGefDeal` for future use?

Reply:
good shout - it's updating the entire deal. I've changed this